virttest/libvirt_xml/devices/vsock.py

```diff
@@ -10,7 +10,7 @@
 
 class Vsock(base.UntypedDeviceBase):
 
-    __slots__ = ('model_type', 'cid', 'address', 'alias')
+    __slots__ = ('model_type', 'cid', 'address', 'alias', 'address')
```

Comment:
'address' is already in __slots__. Does it added by mistake?

Reply:
@Yingshun Ah, yes, thanks!

Reply:
@Yingshun Updated.